### PR TITLE
improve docs and style

### PR DESCRIPTION
**Pull request by arthur-cw**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 on:
   push:
     branches: [ "main" ]
   pull_request:
-    branches: [ "main" ]
+    branches: "**"
 
 jobs:
-  compile_contracts:
+  ubuntu-latest:
```
Comment (EvgeniiZaitsevCW) [inline .github/workflows/build.yml:13]: Replace "ubuntu-latest" => "compile_contracts"

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+name: test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: "**"
+
+jobs:
+  ubuntu-latest:
```
Comment (EvgeniiZaitsevCW) [inline .github/workflows/test.yml:13]: Rename "ubuntu-latest" => "run_contracts_tests"

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-name: Node.js CI
+name: build
 
 on:
   push:
     branches: [ "main" ]
   pull_request:
-    branches: [ "main" ]
+    branches: "**"
```
Comment (EvgeniiZaitsevCW) [inline .github/workflows/build.yml:10]: Here and in the same place of the `test.yaml` file.
Maybe it is better to leave only `main` branch here.

**Review by EvgeniiZaitsevCW — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (arthur-cw) [reply .github/workflows/build.yml:13]: we want the name of the job to show the system :-)

Comment (arthur-cw) [reply .github/workflows/build.yml:13]: image

Comment (EvgeniiZaitsevCW) [reply .github/workflows/build.yml:13]: Got it! Thanks

File: .github/workflows/build.yml (hunk 3 above)
Comment (EvgeniiZaitsevCW) [reply .github/workflows/build.yml:10]: Ok. We can change it later.

**Review by EvgeniiZaitsevCW — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -9,4 +9,7 @@ cache
 artifacts
 
 #IDE files
-/.idea/
\ No newline at end of file
+/.idea/
+
+# OpenZeppelin
+.openzeppelin/unknown-*.json
```
Comment (EvgeniiZaitsevCW) [inline .gitignore:15]: I don't think it is a good idea.
We at least need to save `.openzeppelin/unknown-2009.json` for our `Mainnet`.

**Review by EvgeniiZaitsevCW — APPROVED** [review]

File: .gitignore (hunk 4 above)
Comment (EvgeniiZaitsevCW) [inline .gitignore:15]: I don't think it is a good idea.
We at least need to save `.openzeppelin/unknown-2009.json` for our `Mainnet`.

**Review by EvgeniiZaitsevCW — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (arthur-cw) [reply .gitignore:15]: maybe we should change the file's name :thinking:  usually files with the `unknown` substring are used for development, check [here](https://docs.openzeppelin.com/upgrades-plugins/1.x/network-files#configuration-files-in-version-control)

Comment (EvgeniiZaitsevCW) [reply .gitignore:15]: I agree

**Review by EvgeniiZaitsevCW — APPROVED** [review]

**Review by igorsenych-cw — APPROVED** [review]

**Pull request merged**